src/components/MessageAttachments.vue

```diff
@@ -118,6 +118,7 @@ export default {
 		downloadZip() {
 			window.open(this.zipUrl)
 			window.focus()
+			window.location = this.url
```

Comment:
```suggestion
			window.location = this.zipUrl
```

right?

Reply:
yes of course. ctrl+z did me dirty :laughing: